### PR TITLE
Added environment injection to RoyalRender rendering jobs

### PR DESCRIPTION
**Pull request by kalisp**
Removed obsolete OP jobs, only pre job script necessary.

Direct copy from customer site, where it is used in production.

### CODE REVIEW

File: client/ayon_royalrender/rr_root/render_apps/_prepost_scripts/Ayon_ayon_inject_envvar.cfg
Comment (antirotor) [inline client/ayon_royalrender/rr_root/render_apps/_prepost_scripts/Ayon_ayon_inject_envvar.cfg:1]: comment for this var probably doesn't reflect what it does

**Review by antirotor — DISMISSED** [review]
I can't currently test it because of the licensing issues, but code looks good and if it is already used in production without issues, I guess it can be safely merged. Just those two comments.

File: client/ayon_royalrender/lib.py [hunk 1]
```diff
@@ -12,25 +12,29 @@
 from ayon_core.lib import (
     BoolDef,
     NumberDef,
+    is_in_tests,
```
[the remark this answers is not in the document]
Comment (kalisp) [reply client/ayon_royalrender/lib.py:12]: Cleaned up imports.

File: client/ayon_royalrender/rr_root/render_apps/_prepost_scripts/Ayon_ayon_inject_envvar.cfg
Comment (kalisp) [reply client/ayon_royalrender/rr_root/render_apps/_prepost_scripts/Ayon_ayon_inject_envvar.cfg:1]: Updated docstring

**Pull request merged**